### ISSUE TITLE
unable to successfully run yarn ios

### ISSUE THREAD

**Issue by benjaminharringtonrose** [cawfree/create-react-native-dapp#26]
- **I'm submitting a ...**
  [ x] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**

node - 14.17.3
expo-cli - 4.13.0
yarn - 1.22.10

"dependencies": {
  "expo": "^43.0.0",
  "react": "^17.0.1",
  "react-dom": "^17.0.1",
  "react-native": "0.64.2",
  "react-native-web": "~0.17.1",
  "@react-native-async-storage/async-storage": "1.13.4",
  "@walletconnect/react-native-dapp": "1.6.1",
  "react-native-svg": "9.6.4",
  "base-64": "1.0.0",
  "buffer": "6.0.3",
  "node-libs-browser": "2.2.1",
  "path-browserify": "0.0.0",
  "react-native-crypto": "2.2.0",
  "react-native-dotenv": "2.4.3",
  "react-native-localhost": "1.0.0",
  "react-native-get-random-values": "1.5.0",
  "react-native-stream": "0.1.9",
  "web3": "1.3.1"
},

My issue is when I run `npx create-react-native-dapp`, and then run `yarn ios` (which does indeed run the script), it seems that expo gets hung up on asking be about my bundle identifier (which is weird, I think it's trying to eject it). My output looks like this:

dapp % yarn ios
yarn run v1.22.10
$ node_modules/.bin/ts-node scripts/ios
npx: installed 3 in 1.392s
Process on port 8545 killed
Nothing to compile

📝  iOS Bundle Identifier Learn more: https://expo.fyi/bundle-identifier

? What would you like your iOS bundle identifier to be? › Nothing to compile
Started HTTP and WebSocket JSON-RPC server at http://0.0.0.0:8545/

Accounts
========
Account #0: 0xa7546dc1513578d447d7b702fea3ddb4f9927bf9 (1000 ETH)
Private Key: 0x562b680d61dbfcf946bdb4086283c70b7aa7a2536f877015c575c67a32990afb

Account #1: 0x29b0f31703fe7c83846ed745010d9bce316fa585 (1000 ETH)
Private Key: 0x7fa84b169a57229ab3da8b325c685a63acf9e603a7d074f7d4a1f6a9ae9628cd

Account #2: 0x3846b2d7e22661f7dbff00419840e8c94b2b9dc1 (1000 ETH)
Private Key: 0x3bf5516b5c1fee47034e5152df29a5c4b8b2a12fadee0cd10df0b2c6d28d5437

Account #3: 0x883b8ef4ebe508493ada10118832aeb76d63a77b (1000 ETH)
Private Key: 0xe873f18421316bb06334e85199c0c75cc5cd279924c77925e0a80b778c163f1c

Account #4: 0x7bdbccb1c7ee518810aabf6dc94d23c2b36794e2 (1000 ETH)
Private Key: 0x0453e909ffa458027dfff12c13e203c696e8e78b046c45c2748256fa95cae60b

Account #5: 0xed26718815d2ca7111602a0d43e20229a8b42d7f (1000 ETH)
Private Key: 0x02bc7bdd5717042bdcbfc8683f73e7e3e46209eeca0823668ff580bf59054b56

Account #6: 0x8302fb751c2933b63d6f73f6e5e2564651c095c8 (1000 ETH)
Private Key: 0x42dde9b5a3a5a007e8b8ed4229707ffaee57f3b538e440fc00192199b5847362

Account #7: 0xafa1c6276c1d97c0e939afc4605c6629332a3679 (1000 ETH)
Private Key: 0xdb9649a3f752cee8d269b31e6903d1cda701c51fcdea2780d6b5d10ccf3f8e5e

Account #8: 0xd807def0eb395c26b029290231bb8d9795be41e7 (1000 ETH)
Private Key: 0xdcb2c2921b7fc57af7695f8f4fc1ee5631c387557fc097f7fef30242e558a0c3

Account #9: 0xdc46a6aeef68064986026de2a1a8dbf39ed9535b (1000 ETH)
Private Key: 0x7766d57d00db3a5b927704840bb9f34a644e9b32ba2f1091cec2f9fe1c1253ee

So at the point of "What would you like your iOS bundle identifier to be?" I'm guessing it's getting hung up. So I ran `expo run:ios` and it ejected my ios project into ios folder, installed pods, and built a stand-alone app. I'm unsure if this is related to this project, some new change I'm unaware of with expo , or versioning issues, but any help would be greatly appreciated!!! I LOVE what you've created so far. Super super cool. Thanks!!!

- **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)

**Comment by benjaminharringtonrose** [comment]
Ahh I see it's supposed to be ejected! Sorry!!!

**Comment by benjaminharringtonrose** [comment]
Just to document what I did to get it working, I ran `expo run:ios`, answered the bundle identifier question, then let it build. I then had to go back and replace the overwritten scripts in `package.json` with

`"android": "node_modules/.bin/ts-node scripts/android",`
`"ios": "node_modules/.bin/ts-node scripts/ios",`

and then ran `yarn ios`. Then everything worked like magic! Thanks again cawfree!!!!
